packages/validation/docs/core-plugins.md

```diff
@@ -1,80 +1,111 @@
 ## Core Validation plugins
 
-There are four cores plugins:
+There are several cores plugins:
 
-- [Open Policy agent](#open-policy-agent)
-- [Kubernetes Schema](#kubernetes-schema)
+- [Kubernetes Schema](#kubernetes-schema) - includes deprecation/violation 
```

Comment:
Maybe here, we can duplicate plugins descriptions from https://github.com/kubeshop/monokle-core/pull/409/files#diff-d5870eae7fbe90f181dbda208cc45765807cebe168d9b7cbf2cf3d497e28960cR30-R36?